pkg/cmd/pipeline/start.go

```diff
@@ -188,15 +189,34 @@ func (opt *startOptions) getInput(pname string) error {
 }
 
 func (opt *startOptions) getInputResources(resources resourceOptionsFilter, pipeline *v1alpha1.Pipeline) error {
-	var ans string
 	for _, res := range pipeline.Spec.Resources {
 		options := getOptionsByType(resources, string(res.Type))
+		// directly create resource
 		if len(options) == 0 {
-			return fmt.Errorf("no pipeline resource of type %s found in namespace: %s",
-				string(res.Type), opt.cliparams.Namespace())
+			ns := opt.cliparams.Namespace()
+			fmt.Fprintf(opt.stream.Out, "no pipeline resource of type \"%s\" found in namespace: %s\n", string(res.Type), ns)
+			fmt.Fprintf(opt.stream.Out, "please create new \"%s\" resource\n", string(res.Type))
+			newres, err := opt.createPipelineResource(res.Name, res.Type)
+			if err != nil {
+				return err
+			}
+			fmt.Printf("resource status %s\n\n", newres.Status)
+			opt.Resources = append(opt.Resources, res.Name+"="+newres.Name)
+			return nil
+		}
+		// first ask if user wants to create new resource or use existing one
```

Comment:
Should we do that (ask the question first), or should we had an "create a resource" option in the list ? (the later seems tricker to achieve though :sweat:)

Reply:
@vdemeester 
@sthaha also suggests the same thing.
It was not that tricky as we thought of, I have updated the PR addressing the same.